koku/api/report/ocp/query_handler.py

```diff
@@ -291,6 +291,11 @@ def execute_query(self):  # noqa: C901
         }
         ordered_total.update(query_sum)
 
+        if self.is_csv_output:
+            if self._report_type in ("virtual_machines"):
+                # Handle formating OCP VM response
+                data = self.format_vm_csv_response(query_data)
```

Comment:
What is the intention with this:
```
if self._report_type in ("virtual_machines")
```
?

We can also consolidate this logic:
```suggestion
        if self.is_csv_output and self._report_type == "virtual_machines":
            # Handle formating OCP VM response
            data = self.format_vm_csv_response(query_data)
```

Reply:
So originally I had this in a common place for consolidate EC2 and OCP VM's but decided not to do that now. That did mean I left ugly style logic when I moved it, good catch. Though with Cody's point to we should probably just move this up a little.